gitignore

```diff
@@ -0,0 +1,56 @@
+.gradle
```

Comment:
rename file "gitignore" to ".gitignore"

Reply:
also please remove "lib" and "build" folders from PR

Reply:
made: rename file "gitignore" to ".gitignore", remove "lib", "build", ".gradle"